src/js/engagement_view/src/components/engagementView/sidebar/tables/toggleLensTable.tsx

```diff
@@ -62,7 +62,7 @@ export function ToggleLensTable({ setLens }: ToggleLensTableProps) {
 					}
 				}
 			);
-		}, 1000);
+		}, 5000);
```

Comment:
this is an annoyance I've wanted to change for a while - the constant pinging of the graphql endpoint is slowed down a bit.